components/cluster/service-hacluster/files/svc-hacluster

```diff
@@ -16,7 +16,7 @@ export CLUSTER_USER=hacluster
 export CLUSTER_GROUP=haclient
 export PATH=/usr/gnu/bin:${PATH}
 MYNAME=hacluster
-APPPATH=/usr/sbin/
+APPPATH=/usr/sbin/amd64/
```

Comment:
If we ship 64-bit binaries in 32-bit paths like we do for 64-bit only components, this change is not needed. Shipping 64-bit binaries in 32-bit paths is the preferred solution.